extensions/mssql/src/contracts.ts

```diff
@@ -1114,3 +1114,16 @@ export namespace DisposeTableDesignerRequest {
 	export const type = new RequestType<azdata.designers.TableInfo, void, void, void>('tabledesigner/dispose');
 }
 // ------------------------------- < Table Designer > ------------------------------------
+
+
+// ------------------------------- < Execution Plan > ------------------------------------
+
+export interface GetExecutionPlanParams {
+	graphInfo: azdata.executionPlan.ExecutionPlanGraphInfo,
+}
+
+export namespace GetExecutionPlanRequest {
+	export const type = new RequestType<GetExecutionPlanParams, azdata.executionPlan.GetExecutionPlanResult, void, void>('queryexecutionplan/getexecutionplan');
```

Comment:
Easier to read if you camelCase method names (e.g. `queryExecutionPlan/getExecutionPlan`)

Reply:
Originally the guidance was to have all our DMP method names all lower-case, since that is what vscode was doing (IIRC).  I'd don't think it matters too much as this is a bit of an implementation detail, but there is some value in being consistent.